app/main.py

```diff
@@ -1,3 +1,63 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: (int, float)) -> None:
```

Comment:
The type hint for the `km` parameter should be `Union[int, float]` instead of `(int, float)` to properly indicate that it can be either an integer or a float.